### PR TITLE
#159467252 Make the login page mobile responsive (update)

### PR DESCRIPTION
**Pull request by Spectrumsun**
#### What does this PR do?
- Make sure the login page is responsive
- Make sure a grid system is used

#### Description of Task to be completed?
- When the user visits the site the login page should adjust to the display of the display
- The login page should be responsive
- The login page should use a grid system

#### How should this be manually tested?
* Clone the repository
* Cd to the project folder
* Run `chmod +x docker.sh` on the terminal
* Start the server using the following commands
    * `./docker.sh development` (for development purposes)  
    *    `./docker.sh production` (for production purposes)
* On the browser, navigate to localhost:3000 for development purposes

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#159467252](https://www.pivotaltracker.com/n/projects/2184887/stories/159467324)

#### Screenshots (if appropriate)
screen shot 2018-08-14 at 11 01 55 2
screen shot 2018-08-14 at 11 01 45 2

#### Questions:
None

### CODE REVIEW

File: src/views/Login/Login.scss [hunk 1]
```diff
@@ -37,6 +37,7 @@
     border-radius: 4px;
     background-color: #3359db !important;
     margin-left: 80px;
+    margin-bottom: 250px;
```
Comment (albert85) [inline src/views/Login/Login.scss]: Please change this margin format to `margin: 0px 0px 250px 80px`

File: src/views/Login/Login.scss [hunk 2]
```diff
@@ -72,73 +66,131 @@
...
+
+    &__travel-request-text {
+      padding: 100px 30px 0px;
+      font-size: 35px;
+    }
+
+    &__login-btn {
+      width: 200px;
+      margin-left: 30px;
+      margin-bottom: 20px;
```
Comment (albert85) [inline src/views/Login/Login.scss]: Please change this format to the one i indicated above

File: src/views/Login/Login.scss [hunk 3]
```diff
@@ -72,73 +66,131 @@
...
+    }
+
+    &__login-to-get-started-text {
+      font-size: 13px;
+    }
+
+    &__link-text {
+      font-size: 15px;
+      padding-left: 0px;
+      padding-bottom: 100px;
```
Comment (albert85) [inline src/views/Login/Login.scss]: use this `padding: 0px 0px 100px 0px` or delete padding-left: 0px if it won't affect the position of the text link

File: src/views/Login/Login.jsx [hunk 4]
```diff
@@ -34,6 +34,7 @@ export class Login extends Component {
     window.location.replace(url);
```
Comment (albert85) [inline src/views/Login/Login.jsx]: the branch name is more than 3 words according to style guide use `ft-login-page-responsive-155895875` will do i.e. ft-{2 or 3 words}-PT(ID)

Comment (Spectrumsun) [reply src/views/Login/Login.jsx]: Thanks for the feedback have updated with the changes. Due to changing the name of the branch, the PR had to be closed. Please check the new PR raised for any other feedback. Thanks, @albert85  @samuelkubai

**Pull request closed without merging**